pkg/appmanager/resourceConfig.go

```diff
@@ -813,6 +814,35 @@ func setProfilesForMode(mode string, cfg *ResourceConfig) {
 	}
 }
 
+func verifySourceAddrTranslation(vsName string, sat SourceAddrTranslation) error {
+	if sat.Type == "none" && sat.Pool != "" || sat.Type == DEFAULT_SOURCE_ADDR_TRANSLATION && sat.Pool != "" {
```

Comment:
Use parentheses to make this clearer even though it's logically correct as is (to separate the and/or clauses).  Also, why is there a definition for "automap" but not for "none"? I'd create constants for all three possibilities (none, automap, snat).  There are idiomatic ways to create enum-like constants in GO.  You can then set the default to the automap contstant.

Reply:
Wouldn't the logic
```
if sat.Type != "snat" && sat.Pool != "" {
```
be simpler?

Reply:
Yes it would fixing.